src/scale.c

```diff
@@ -180,3 +182,33 @@ avifBool avifImageScale(avifImage * image,
 }
 
 #endif
+
+avifResult avifImageCopySamples(avifImage * dstImage, const avifImage * srcImage, avifPlanesFlags planes)
```

Comment:
Define `avifImageCopySamples()` in src/avif.c. This function is similar to `avifImageCopy()`, so they should be defined in the same file. This function is unrelated to scaling an image.